Dockerfile

```diff
@@ -1,3 +1,28 @@
 FROM jenkinsci/jnlp-slave
```

Comment:
There is no such version for jenkins slave.
https://hub.docker.com/r/jenkinsci/jnlp-slave/tags/
If we will hit any issues with versioning - 
let's use ssh node connection (slave.jar copy from master node on startup)
or build or made our slave image fat and use version we need.